d2bs/kolbot/libs/bots/Diablo.js

```diff
@@ -429,16 +429,33 @@ function Diablo() {
 	if (me.area !== 107) {
 		Pather.useWaypoint(107);
 	}
-
-	if (!Pather.moveTo(7790, 5544)) {
-		throw new Error("Failed to move to Chaos Sanctuary");
+//Check Config.Diablo.ClearPath option for true or false and clear the path if walking.
```

Comment:
don't think the say is necessary, the throw error string you have is wrong, logic same as above change to this:
```
if (!Pather.moveTo(7790, 5544, 5, Config.Diablo.ClearPath)) {
	throw new Error("Failed to move to Chaos Sanctuary");
}
```